eva_cttv_pipeline/evidence_strings.py

```diff
@@ -47,25 +47,27 @@ def get_cttv_variant_type(ref, alt):
 
 
 class CTTVEvidenceString(dict):
-    def __init__(self, a_dictionary, ensembl_gene_id=None, clinvar_record=None, ensembl_gene_id_uri=None, clin_sig=None,
-                 efo_list=None, ref_list=None):
+    # def __init__(self, a_dictionary, ensembl_gene_id=None, clinvar_record=None, clin_sig=None, efo_list=None,
+    #              ref_list=None, report=None):
+    def __init__(self, a_dictionary, record=None, efo_list=None, ref_list=None, ensembl_gene_id=None, report=None):
         super().__init__(a_dictionary)
         # dict.__init__(a_dictionary)
 
         if ensembl_gene_id:
             self.add_unique_association_field('gene', ensembl_gene_id)
-        if clinvar_record:
-            self.add_unique_association_field('clinvarAccession', clinvar_record.accession)
+        if record:
+            self.add_unique_association_field('clinvarAccession', record.clinvarRecord.accession)
 
-        if ensembl_gene_id_uri:
+        if ensembl_gene_id:
+            ensembl_gene_id_uri = get_ensembl_gene_id_uri(ensembl_gene_id)
             try:
-                self.set_target(ensembl_gene_id_uri, clin_sig_2_activity[clin_sig])
+                self.set_target(ensembl_gene_id_uri, clin_sig_2_activity[record.clin_sig])
             except KeyError:
-                # unrecognised_clin_sigs.add(clin_sig)  # TODO fix this
+                report.unrecognised_clin_sigs.add(record.clin_sig)  # TODO fix this
                 self.set_target(ensembl_gene_id_uri, 'http://identifiers.org/cttv.activity/unknown')
 
-        if ref_list and len(ref_list) > 0:
-            self.top_level_literature = ref_list
+            if ref_list and len(ref_list) > 0:
+                self.top_level_literature = ref_list
```

Comment:
Just in case it was caused by a bogus formatting command, is this extra indentation expected?